identity.md

```diff
@@ -0,0 +1,70 @@
+
+# 👤 Identity 
+
+in KILT, an identity is an object representing an entity - for example a person, an organization or even a machine.     
+
+An identity contains multiple properties.  
+One of them is the `address` : it's the entity's unique and public identifier.     
+
+> 💡 Asymmetric cryptography  
+> If you're interested in the cryptographic details, a KILT identity contains:
+> * An encryption keypair (x25519-xsalsa20-poly1305) used to encrypting messages between participants of the system;   
+> * A signing keypair (ed25519, will be replaced with sr25519);
+> * The address, which is generated from the signing public key using the ss58 algorithm. 
+
+All we need to create an identity is a mnemonic.   
+
+> 💡 Mnemonics   
+> In cryptography, a mnemonic is a random series of words. `gold upset segment cake universe` is a mnemonic. It's used to generate a public address (via a key). What's great about a mnemonic is that it's human-readable, so a person can simply remember it to re-generate their keypairs and address later. 
+
+## Code 
+
+To generate an identity, two methods from the KILT SDK are needed: 
+* `generateMnemonic`
+* `buildFromMnemonic` // takes a mnemonic as an input, and outputs an `Identity` instance.  
+
+Create a new file `1-generateMnemonic.js`.
+
+Open it and paste the following code: 
+
+```javascript
+// import KILT SDK
+const Kilt = require('@kiltprotocol/sdk-js')
+
+const mnemonic = Kilt.Identity.generateMnemonic()
+console.log('mnemonic:', mnemonic)
+
+const identity = Kilt.Identity.buildFromMnemonic(mnemonic)
+console.log('address:', identity.address)
+```
+
+You're now ready to generate an Identity.    
+
+## Run 
+
+To generate an identity, run this command in your terminal (still within your `kilt-rocks` directory):  
+```bash
+node 1-generateMnemonic.js
+``` 
+
+Your output should look like this (but not be identical to it, since the mnemonic is randomly generated):
+```bash
+mnemonic: gold upset segment cake universe carry demand comfort dawn invite element capital
+address: 5CUoo2vAegeaZHPNdxZyuMesR3RWYBKHj4jfVyj4FXzpXPuR
+```     
+
+You want to run this command three times, in order to generate three identities.   
+Each time, copy/paste the output (mnemonic and address) somewhere since we'll need these identities later.  
+
+In the next steps, we'll refer to the so-generated identities as follows:  
+* `[claimer mnemonic]` is the mnemonic you've generated on the first run of the command above, and `[claimer address]` the associated address;  
+* `[attester mnemonic]` is the mnemonic you've generated on the second run of the command above, and `[attester address]` the associated address;      
+* `[verifier mnemonic]` is the mnemonic you've generated on the second run of the command above, and `[verifier address]` the associated address.
+
+That's it - You've successfully generated three new identities and their address!  
+
```

Comment:
Awesome! Only the general about the code needing more spacing and indentations 💯